src/content/consul/product-landing.json

```diff
@@ -22,7 +22,7 @@
 				"url": "/consul/tutorials/gs-consul-service-mesh/get-started"
 			},
 			{
-				"icon": "server-24",
+				"icon": "server",
```

Comment:
This change checks out ✅ 

![image](https://user-images.githubusercontent.com/43934258/190218925-beca8b71-0424-43dd-b2be-f9c3ab1c3d49.png)